### PR TITLE
add quoted atoms to s-expression parser

### PR DESCRIPTION
**Pull request by jannikvierling**

### CODE REVIEW

File: core/src/main/scala/at/logic/gapt/formats/lisp/SExpression.scala [hunk 1]
```diff
@@ -66,8 +66,9 @@ class SExpressionParser( val input: ParserInput ) extends Parser {
 
   def Str = rule { '"' ~ capture( zeroOrMore( noneOf( "\"" ) ) ) ~ '"' ~ WhiteSpace ~> lisp.LAtom }
   def Atom = rule { capture( oneOrMore( noneOf( "() \n\r\t\f;\"" ) ) ) ~ WhiteSpace ~> lisp.LAtom }
+  def QuotedAtom = rule { "|" ~ capture( oneOrMore( noneOf( "\\|" ) ) ) ~ "|" ~ WhiteSpace ~> lisp.LAtom }
```
Comment (gebner) [inline core/src/main/scala/at/logic/gapt/formats/lisp/SExpression.scala]: So `|\||` is not a quoted atom?  [The "official" grammar allows it.](https://github.com/tip-org/tools/blob/master/tip-lib/src/TIP.bnfc#L117)

Not that it matters, I just got curious when you explicitly excluded the backslash.

Comment (jannikvierling) [reply core/src/main/scala/at/logic/gapt/formats/lisp/SExpression.scala]: Ok, then I think we should allow it too.

**Comment by gebner** [comment]
Why is this not merged yet?

**Comment by gebner** [comment]
BTW, this PR broke the user manual again.

**Pull request merged**